src/nft/components/profile/view/ProfilePage.tsx

```diff
@@ -179,9 +181,10 @@ export const ProfilePage = () => {
             onClick={() => setSellPageState(ProfilePageStateType.LISTING)}
```

Comment:
this functionality should be switched to toggleBag instead of go straight to list